src/state/bulk-server-url-state.ts

```diff
@@ -0,0 +1,13 @@
+import { atom } from 'recoil';
+import { recoilPersist } from 'recoil-persist';
+
+const { persistAtom } = recoilPersist();
+
+/*
+ * State for managing the bulk export server url
+ */
+export const bulkServerURLState = atom<string | null>({
```

Comment:
```suggestion
export const bulkServerURLState = atom({
```
I think this is safe enough: https://gist.github.com/mgramigna/bdc933fae2150216681c28d1f0f7aac7#recoil

Reply:
When I changed it to look like this like the style guide says, typescript couldn't infer the type? I am not sure why.

`const exampleAtom = atom({
  key: "exampleAtom",
  default: "",
});`

Reply:
When are you getting this error?